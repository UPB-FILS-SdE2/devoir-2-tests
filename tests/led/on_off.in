led on 0 0
led on 0 1
led off 0 0
quit