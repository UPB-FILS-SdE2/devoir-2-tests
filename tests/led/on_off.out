cmd: Set pixel 0 0 to 9
cmd: Set pixel 0 1 to 9
cmd: Set pixel 0 0 to 0
cmd: 